packages/terra-clinical-result/src/terra-dev-site/test/clinical-result/flowsheet-result-cell/TruncatedFlowsheetResultCell.test.jsx

```diff
@@ -141,19 +185,25 @@ export default () => (
       <div className={cx('mock-flowsheet-resultcolumn-cell')}>
         <FlowsheetResultCell resultDataSet={multipleResultValues3} />
       </div>
+      <div className={cx('mock-flowsheet-resultcolumn-cell')}>
+        <FlowsheetResultCell resultDataSet={testExtremeWidthBloodPressure} hideUnit />
+      </div>
     </div>
     <div className={cx(['mock-flowsheet-resultcolumn', 'half-column'])}>
       <div className={cx('mock-flowsheet-resultcolumn-cell')}>
-        <FlowsheetResultCell resultDataSet={multipleResultValues} />
+        <FlowsheetResultCell resultDataSet={multipleResultValues} className={cx('flowsheetresult-minwidth-override--for-testing-only')} />
```

Comment:
This new sequence is probably better updating test like this:
https://gist.github.com/neilpfeiffer/62248d3589e2b965e1a5282da7dbc303

Removes the minwidth-override (since there is no reason to test that column size any longer), adds an empty column beside to show the overflow, and an extra message in the last cell on the the last row.

Reply:
Fixed here https://github.com/cerner/terra-clinical/pull/637/commits/788a6eed719217b512d85dbc8c1fa1c37d747e50#diff-3409108aa132fdc7b0380a3b5a62bbf5R194